.env

```
@@ -0,0 +1,23 @@
+PORT=5001
+MongoDB_URI=mongodb+srv://fsdteamsaa:DuxRuAdJ4owB7LHs@cluster0.05g37mt.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The MongoDB_URI key should be in uppercase

(LowercaseKey)